avocado/utils/pci.py

```diff
@@ -59,6 +59,22 @@ def get_pci_addresses():
         return addresses
 
 
+def get_pci_hbas():
+    """
+    Gets list of PCI hbas, first pci address in a hba, in the system.
+    Does not return the PCI Bridges/Switches.
+    e.g: for pci address 0000:00:14.0 and 0000:00:14.1, we return 0000:00:14.0
+
+    :return: list of full PCI hbas including domain (0000:00:14.0)
+    """
```

Comment:
Well IMO this does not require a special function, one simply does `set(_[:-2] for _ in get_pci_addresses())`. Anyway if you insist I'd call such function `get_occupied_devices` or similar as it reports `bus-device` and omits `function` (or am I wrong? Not really expert about PCI addressing).